src/bufr/BufrParser/Exports/Variables/SpectralRadianceVariable.cpp

```diff
@@ -0,0 +1,157 @@
+/*
+ * (C) Copyright 2022 NOAA/NWS/NCEP/EMC
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include <bits/stdc++.h>
```

Comment:
This include should be avoided - it's not guaranteed to be portable across different platforms/compilers. See https://stackoverflow.com/questions/31816095/why-should-i-not-include-bits-stdc-h for details. Another issue is that it includes every STL include file which is way more than you typically need, slows down compile time and potentially increases the executable size.

Reply:
@srherbener Thanks for the advise.  I have removed #include <bits/stdc++.h>` from the code.
Ctest passed.

Reply:
@emilyhcliu have you pushed your changes to github? The `#include <bits/stdc++.h>` appears to be still here.

Reply:
@srherbener Sorry, I did not push.  I just did the push.  Code is updated now.

Reply:
@emilyhcliu no problem. Thanks for pushing to github. 

When I merged develop a couple hours ago, I accidentally created a syntax error of which I have just fixed. Sorry about that. The build should work now, and hopefully all the tests will pass.

Reply:
@srherbener, no worries.  Thanks for your review.